docs/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
   - Implement DjangoQL for Flaw filtering (OSIDB-3337)
+  - Set requires_cve_description to REQUESTED when unset and the flaw
+    has cve_description (OSIDB-3349)
```

Comment:
Could you please rebase changelog? Thanks!

Reply:
done!

Reply:
Oh, I accidentally put on extra space before the `-` character when updating changelog: ` - Implement DjangoQL for Flaw filtering (OSIDB-3337)`. Could you please fix it? Thanks :D

Reply:
Fixed!